libraries/AP_Scripting/AP_Scripting.cpp

```diff
@@ -54,6 +54,14 @@ const AP_Param::GroupInfo AP_Scripting::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("VM_I_COUNT", 2, AP_Scripting, _script_vm_exec_count, 10000),
 
+    // @Param: HEAP_SIZE
+    // @DisplayName: Scripting Heap Size
+    // @Description: Amount of memory available for scripting
+    // @Range: 1000 1000000
+    // @Increment: 1024
+    // @User: Advanced
+    AP_GROUPINFO("HEAP_SIZE", 3, AP_Scripting, _script_heap_size, 32*1024),
```

Comment:
32k seems a bit huge for fmuv3. This might be a good place to use a define that is set by different cpu memory architectures/sizes.

Reply:
Does this require a reboot?

Reply:
Does require a reboot. You need a decent amount of memory, once I loaded 2-3 simple scripts I needed more memory (primarily due to the compilation phase).